tests/schema/__snapshots__/atomicOperations.test.ts.snap

```diff
@@ -0,0 +1,373 @@
+
+// Pagination type
+type Pagination = {
+  first?: boolean
+  last?: boolean
+  before?: boolean
+  after?: boolean
+}
+
+// Prisma custom scalar names
+type CustomScalars = 'No custom scalars are used in your Prisma Schema.'
+
+// Prisma model type definitions
+interface PrismaModels {
+  ModelName: Prisma.ModelName
+}
+
+// Prisma input types metadata
+interface NexusPrismaInputs {
+  Query: {
+    modelNames: {
+      filtering: 'AND' | 'OR' | 'NOT' | 'id' | 'name' | 'amount'
+      ordering: 'id' | 'name' | 'amount'
+    }
+  },
+  ModelName: {
+
+  }
+}
+
+// Prisma output types metadata
+interface NexusPrismaOutputs {
+  Query: {
+    modelName: 'ModelName'
+    modelNames: 'ModelName'
+  },
+  Mutation: {
+    createOneModelName: 'ModelName'
+    updateOneModelName: 'ModelName'
+    updateManyModelName: 'BatchPayload'
+    deleteOneModelName: 'ModelName'
+    deleteManyModelName: 'BatchPayload'
+    upsertOneModelName: 'ModelName'
+  },
+  ModelName: {
+    id: 'Int'
+    name: 'String'
+    amount: 'Int'
+  }
+}
+
+// Helper to gather all methods relative to a model
+interface NexusPrismaMethods {
+  ModelName: Typegen.NexusPrismaFields<'ModelName'>
+  Query: Typegen.NexusPrismaFields<'Query'>
+  Mutation: Typegen.NexusPrismaFields<'Mutation'>
+}
+
+interface NexusPrismaGenTypes {
+  inputs: NexusPrismaInputs
+  outputs: NexusPrismaOutputs
+  methods: NexusPrismaMethods
+  models: PrismaModels
+  pagination: Pagination
+  scalars: CustomScalars
+}
+
+declare global {
+  interface NexusPrismaGen extends NexusPrismaGenTypes {}
+
+  type NexusPrisma<
+    TypeName extends string,
+    ModelOrCrud extends 'model' | 'crud'
+  > = Typegen.GetNexusPrisma<TypeName, ModelOrCrud>;
+}
+  "
+`;
+
+exports[`support atomic operations (default): schema 1`] = `
+"type Mutation {
+  createOneModelName(data: ModelNameCreateInput!): ModelName!
+  deleteOneModelName(where: ModelNameWhereUniqueInput!): ModelName
+  updateOneModelName(data: ModelNameUpdateInput!, where: ModelNameWhereUniqueInput!): ModelName
+  upsertOneModelName(where: ModelNameWhereUniqueInput!, create: ModelNameCreateInput!, update: ModelNameUpdateInput!): ModelName!
+  updateManyModelName(data: ModelNameUpdateManyMutationInput!, where: ModelNameWhereInput): BatchPayload!
+}
+
+type ModelName {
+  id: Int!
+}
+
+input ModelNameCreateInput {
+  name: String!
+  amount: Int!
+}
+
+input ModelNameWhereUniqueInput {
+  id: Int
+}
+
+input ModelNameUpdateInput {
+  name: StringFieldUpdateOperationsInput
+  amount: IntFieldUpdateOperationsInput
+}
+
+type BatchPayload {
+  count: Int!
+}
+
+input ModelNameUpdateManyMutationInput {
+  name: StringFieldUpdateOperationsInput
+  amount: IntFieldUpdateOperationsInput
+}
+
+input ModelNameWhereInput {
+  AND: [ModelNameWhereInput!]
+  OR: [ModelNameWhereInput!]
+  NOT: [ModelNameWhereInput!]
+  id: IntFilter
+  name: StringFilter
+  amount: IntFilter
+}
+
+input StringFieldUpdateOperationsInput {
+  set: String
```

Comment:
Where are the other operations like `increment` `decrement` and so on?

Reply:
I don't know why these operations are not generated on my local machine. They are generated when tests are running on github. I'll spend more time to understand what happens.

Reply:
Maybe you have the wrong version of prisma locally?

Reply:
I use yarn --frozen-lockfile. This install prisma 2.9.0.
atomicNumberOperations is still a preview feature in 2.9.0 and have to be explicitely enabled in schema (released in 2.10). This can explain why they doesn't appears in the snapshot.

But when i look at those github logs:
https://github.com/graphql-nexus/nexus-plugin-prisma/pull/899/checks?check_run_id=1326841882
With the same commands, atomic operations seems to be properly generated...
I spend two hours to figure out how it can differs without any result.

Can you try to generate the snapshot on your side?

Reply:
Indeed on my machine the tests as you have them pass:

![image](https://user-images.githubusercontent.com/284476/97649706-5dd52680-1a2e-11eb-81c4-3330ae10c6b3.png)

> atomicNumberOperations is still a preview feature in 2.9.0 and have to be explicitely enabled in schema

Yes that is probably the reason.

@timsuchanek or @Weakky maybe you can give a tip here.

1. Tests use a data model of:

```
        model ModelName {
          id      Int @id @default(autoincrement())
          name    String
          amount  Int
        }
```

2. This is passed to https://github.com/graphql-nexus/nexus-plugin-prisma/blob/master/tests/__utils.ts#L23
3. In turn this calls `PrismaClientGenerator.getDMMF`
4. Which is imported from `const PrismaClientGenerator = require('@prisma/client/generator-build')`

This is a dark area of the API to me, not least because its undocumented and typed as `any`.

Reply:
But since 2.10 Prisma doesn't need preview flags for atomic operations. So the easy thing to do is move to 2.10 now and things should just start working.

Reply:
I can confirm, after a rebase on master (last commit uses prisma 2.10) all is properly generated 👍 
I can force-push this PR (i dont think github will proprely handle it) or open a new one. What do you prefer?

Reply:
Otherwise i can commit here the updated snapshot and the default "squash and merge" will be fine to make it right.

Reply:
I think you should be able to merge master back on this branch without issue.

Also your idea to manually tweak the the snapshots so that they pass on CI is ok with me too :)

Reply:
perfect, i just push the updated snapshot